test/timing_load_creds.c

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2020-2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef _WIN32
+# include <sys/stat.h>
+# include <openssl/pem.h>
+# include <openssl/x509.h>
+# include <openssl/err.h>
+# include <openssl/bio.h>
+# include <../include/internal/e_os.h>
+# if defined(OPENSSL_SYS_UNIX)
+#  include <sys/resource.h>
+# endif
+
+static char *prog;
+
+static void readx509(const char *contents, int size)
+{
+    X509 *x = NULL;
+    BIO *b = BIO_new_mem_buf(contents, size);
+
+    if (b == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    PEM_read_bio_X509(b, &x, 0, NULL);
+    if (x == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    X509_free(x);
+    BIO_free(b);
+}
+
+static void readpkey(const char *contents, int size)
+{
+    BIO *b = BIO_new_mem_buf(contents, size);
+    EVP_PKEY *pkey;
+
+    if (b == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+    pkey = PEM_read_bio_PrivateKey(b, NULL, NULL, NULL);
+    if (pkey == NULL) {
+        ERR_print_errors_fp(stderr);
+        exit(1);
+    }
+
+    EVP_PKEY_free(pkey);
+    BIO_free(b);
+}
+
+static void print(const char *what, struct timeval *tp)
+{
+    printf("%s %d sec %d microsec\n", what, (int)tp->tv_sec, (int)tp->tv_usec);
+}
+
+static void usage(void)
+{
+    fprintf(stderr, "Usage: %s [flags] pem-file\n", prog);
+    fprintf(stderr, "Flags:\n");
+    fprintf(stderr, "  -c #  Repeat count\n");
+    fprintf(stderr, "  -d    Debugging output (minimal)\n");
+    fprintf(stderr, "  -w<T> What to load T is a single character:\n");
+    fprintf(stderr, "          c for cert (default)\n");
```

Comment:
Based on the code below, `c` is not the default; If no additional value is present, or it's missing, then `usage()` is called.
I'd suggest just removing the "(default)". 
To clarify, I see that if this is NOT specified, then `-wc` is the default, but this implies to me that if just `-w` is supplied without anything else, then `c` would be the default. It would be more appropriate to state "-wc is the default" or similar.

Reply:
Good point that the guidance was slightly misleading. Changed to make clear that `-wc` is the default.